internal/api/schema.go

```diff
@@ -24,7 +25,7 @@ func schemaToProto(s schema.Schema) *stencilv1beta1.Schema {
 
 func (a *API) CreateSchema(ctx context.Context, in *stencilv1beta1.CreateSchemaRequest) (*stencilv1beta1.CreateSchemaResponse, error) {
 	metadata := &schema.Metadata{Format: in.GetFormat().String(), Compatibility: in.GetCompatibility().String()}
-	sc, err := a.schema.Create(ctx, in.NamespaceId, in.SchemaId, metadata, in.GetData())
+	sc, err := a.schema.Create(ctx, in.NamespaceId, in.SchemaId, metadata, in.GetData(), "")
```

Comment:
why empty string?

Reply:
Its an RPC. RPC does not have that new key because its referring to old package. After migration of package it would be fixed also.